modules/python/python-grammar.ym

```diff
@@ -52,6 +56,7 @@
 
 %token KW_PYTHON
 %token KW_PYTHON_FETCHER
+%token KW_PYTHON_HEADER
```

Comment:
this should probably be KW_PYTHON_HTTP_HEADER as this plugin type is HTTP specific.